<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bom</class>
 <widget class="QWidget" name="Bom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
       <item>
        <widget class="SearchLineEdit" name="bomInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="bomRefresh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ListWidget" name="bomList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
       <item>
        <widget class="PushButton" name="bomNew">
         <property name="text">
          <string>新建</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PushButton" name="bomRemove">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="bomUpdate">
         <property name="text">
          <string>同步</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="VerticalSeparator" name="VerticalSeparator"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="TableWidget" name="bomTable">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>bom层级</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>零件名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>父零件ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>零件数量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>零件成本</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>零件生产周期</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>是否外购</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>注释</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="partNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="partRemove">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="openLine">
   <property name="text">
    <string>打开工艺路线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
